Editor/UI/ShaderVariantsWindow.cs

```diff
@@ -14,13 +14,18 @@ class ShaderVariantsWindow : AnalysisWindow, IProjectIssueFilter
 Build the project to view the Shader Variants
 ";
 
-        const string k_PlayerLogInfo = @"
+        const string k_PlayerLogInstructions = @"
 To find which shader variants are compiled at runtime, follow these steps:
-- Enable the Log Shader Compilation option (Project Settings => Graphics => Shader Loading)
+- Enable the Log Shader Compilation option
 - Make a Development build
 - Run the build on the target platform
 - Drag & Drop the Player.log file on this window
 ";
+
+        const string k_PlayerLogParsingUnsupported = @"
+To find which shader variants are compiled at runtime, update to 2018 LTS or newer.
```

Comment:
How about adding a bit of explanation about what is compiled shaders, how do they differ from not-compiled, and why the information about compiled shaders is useful? In short, why should I bother?

Reply:
@iurii-pavliuk  Good idea. I updated the instructions. Let me know what you think.